namespaces/live-1.cloud-platform.service.justice.gov.uk/recalldecisions-prototype/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: recalldecisions-prototype-admin
+  namespace: recalldecisions-prototype
+subjects:
+  - kind: Group
+    name: "github:happygeneralist"
```

Comment:
Access is mapped to namespaces by a github team not individual user. Please use or create a new team which as your user as a member.

Reply:
Thanks @sablumiah I've updated all the references now to have the correct team now